pkg/server/handlers/metrics.go

```diff
@@ -153,15 +207,16 @@ func (m metrics) initUserCountMetric() {
 			logger.Logger.Info("Unable to fetch user RoleBindings. skip metric...")
 			return
 		}
-		var userCount float64
+		// Due to performance reasons we will create a slice with a capacity of the double size of the count of rolebindings
+		users := make([]string, 0, 2*len(roleBindings.Items))
```

Comment:
How about using `k8s.io/apimachinery/pkg/util/sets` which already provides convenience functions like `Has` (instead of implementing `contains` ourselves)?

Reply:
Thanks for the hint. Didn't aware of this. Already applied!